middlewares/authn.js

```diff
@@ -80,6 +79,8 @@ module.exports = function (req, res, next) {
         return next();
       }
     );
+
+    return;
```

Comment:
I spent like an hour trying to figure out why I was getting "header already sent" errors 🤦

Reply:
This was an existing error? Sorry!!

Reply:
Sadly, yes. This was a regression introduced way back in #3020:

https://github.com/PrairieLearn/PrairieLearn/pull/3020/files#diff-bfa663cd4ce127397d92741afe999bdfe7f72f95a4cbbaaae20d907ea6522449L60

Reply:
Oh dear. Ah well 😄